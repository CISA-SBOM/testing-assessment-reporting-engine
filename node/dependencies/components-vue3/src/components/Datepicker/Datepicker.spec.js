import { describe, expect, it } from 'vitest'
import { shallowMount } from '@vue/test-utils'
import Component from './Datepicker.vue'

describe('Datepicker', () => {
  it('should match its default snapshot', () => {
    const wrapper = shallowMount(Component, {})
    expect(wrapper.element).toMatchSnapshot()
  })
})
