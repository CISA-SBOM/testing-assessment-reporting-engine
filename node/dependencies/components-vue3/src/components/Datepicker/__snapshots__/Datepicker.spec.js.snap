// Vitest Snapshot v1

exports[`Datepicker > should match its default snapshot 1`] = `
<div
  class="inline-block w-full"
  data-id="sds-datepicker"
>
  <div
    class="inline-block w-full"
  >
    
    <div
      class="flex w-full gap-2"
    >
      <div
        class="w-full input-group"
      >
        <button
          class="input-group-text fill-current"
          tabindex="-1"
          type="button"
        >
          <svg
            class="fill-current w-4 h-4"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M148 288h-40c-6.6 0-12-5.4-12-12v-40c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v40c0 6.6-5.4 12-12 12zm108-12v-40c0-6.6-5.4-12-12-12h-40c-6.6 0-12 5.4-12 12v40c0 6.6 5.4 12 12 12h40c6.6 0 12-5.4 12-12zm96 0v-40c0-6.6-5.4-12-12-12h-40c-6.6 0-12 5.4-12 12v40c0 6.6 5.4 12 12 12h40c6.6 0 12-5.4 12-12zm-96 96v-40c0-6.6-5.4-12-12-12h-40c-6.6 0-12 5.4-12 12v40c0 6.6 5.4 12 12 12h40c6.6 0 12-5.4 12-12zm-96 0v-40c0-6.6-5.4-12-12-12h-40c-6.6 0-12 5.4-12 12v40c0 6.6 5.4 12 12 12h40c6.6 0 12-5.4 12-12zm192 0v-40c0-6.6-5.4-12-12-12h-40c-6.6 0-12 5.4-12 12v40c0 6.6 5.4 12 12 12h40c6.6 0 12-5.4 12-12zm96-260v352c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V112c0-26.5 21.5-48 48-48h48V12c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v52h128V12c0-6.6 5.4-12 12-12h40c6.6 0 12 5.4 12 12v52h48c26.5 0 48 21.5 48 48zm-48 346V160H48v298c0 3.3 2.7 6 6 6h340c3.3 0 6-2.7 6-6z"
            />
          </svg>
        </button>
        <input
          class="form-control"
          pattern="[0-9]{2}/[0-9]{2}/[0-9]{4}"
          placeholder="mm/dd/yyyy"
          title="mm/dd/yyyy"
          type="text"
        />
      </div>
      <!---->
    </div>
    
  </div>
  <!---->
</div>
`;
